qiskit_aqt_provider/aqt_job.py

```diff
@@ -77,6 +78,10 @@ def _build_memory_mapping(self):
             return qu2cl
         qubit_map = {}
         count = 0
+
+        if isinstance(self.qobj[0], QuantumCircuit):
+            self.qobj = self.qobj[0]
```

Comment:
I would probably change this to be

```suggestion
        # If a list of quantum circuits use the first element
        # since we only can have a maximum of a single
        # circuit per job.
        if isinstance(self.qobj, list):
            self.qobj = self.qobj[0]
```

to make it more explicit. I think this will work fine as is because if `self.qobj` is not a list it's a `QuantumCircuit` object and index 0 on that is a tuple of `(instruction, qargs, cargs)` for the first instruction in the circuit, which is not a `QuantumCircuit`. But checking for a list explicitly is a bit more explicit as to what is happening here.